pcdsdevices/areadetector/plugins.py

```diff
@@ -162,3 +168,42 @@ class HDF5Plugin(ophyd.plugins.HDF5Plugin, FilePlugin):
 
 class MagickPlugin(ophyd.plugins.MagickPlugin, FilePlugin):
     pass
+
+
+class HDF5FileStore(FileStoreHDF5IterativeWrite, HDF5Plugin_V31):
```

Comment:
It's good to use the most updated plugin, but do we actually make use of the signals added in the most recent version?  File writing doesn't need it, if I'm not mistaken.

Reply:
Hmm, good point - we should probably use the minimum version class we can...